packages/components/src/progress-bar/styles.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { keyframes } from '@emotion/react';
+
+/**
+ * Internal dependencies
+ */
+import { COLORS, CONFIG } from '../utils';
+
+const animateProgressBar = keyframes( {
+	'0%': {
+		left: '-50%',
+	},
+	'100%': {
+		left: '100%',
+	},
+} );
+
+export const Track = styled.div`
+	position: relative;
+	overflow: hidden;
+	width: 100%;
+	height: ${ CONFIG.borderWidthFocus };
+	background-color: ${ COLORS.gray[ '100' ] };
```

Comment:
The grayscale theming is still a bit experimental, but I'd like to see how it holds up in this very dynamic use case (#53032). The intent behind our `Theme` system is that it will automatically generate the appropriate grayscale colors as long as you provide a single `background` color prop to the `<Theme>` wrapper. In your case, that would be the main background color of the canvas itself.

```suggestion
	background-color: var(
		--wp-components-color-gray-100,
		${ COLORS.gray[ 100 ] }
	);
```

☝️ We haven't encapsulated this entire string into the `COLORS.gray[ 100 ]` variable yet, sorry for the verbosity.

---

We also have a little tool in Storybook to test it out:

https://github.com/WordPress/gutenberg/assets/555336/b6291e13-02d8-49d2-ac86-0e94c9f35408

Reply:
Updated, thanks for the tip!